docs/reference/semaphore-changelog.md

```diff
@@ -4,6 +4,13 @@ Thank you for using Semaphore!
 We continuously deploy changes that improve the product for you.
 This page is updated on a weekly basis.
 
+### Week of March 30, 2020
+
+- Updates to the ubuntu1804 image: 
+    - Aws-cli 1.18.21 -> 1.18.28
+    - Git        2.25 -> 2.26
+    - Heroku   7.39.0 -> 7.39.1
```

Comment:
is everything ok with using spaces above. Aws-cli seems close to 1.18.21 ... while GIt is far far away from  2.25...? @bogyo210

Reply:
The source follows this pattern. The result/preview will not have the spaces
![image](https://user-images.githubusercontent.com/40260639/77654318-97662080-6f79-11ea-818d-853ab9c99c9c.png)

Reply:
👍